projects/openapi-io/src/parser/openapi-sourcemap-parser.ts

```diff
@@ -30,18 +34,16 @@ export async function dereferenceOpenApi(
   path: string,
   options: {
     externalRefHandler?: ExternalRefHandler;
+    externalRefHeaders?: UserDefinedHeadersByUrlPrefix;
   } = {}
 ): Promise<ParseOpenAPIResult> {
   const resolver = new $RefParser();
 
   const sourcemap = new JsonSchemaSourcemap(path);
   const resolve = {
     file: options.externalRefHandler,
-    http: {
```

Comment:
Substituted out the `http` handler for our own custom http handler so we can send different headers to different urls